code/game/objects/effects/spawners/lootdrop.dm

```diff
@@ -158,7 +158,7 @@
 				/obj/item/storage/secure/briefcase/syndie = 2,
 				/obj/item/storage/fancy/cigarettes/cigpack_syndicate = 2,
 				/obj/item/storage/pill_bottle/fakedeath = 2,
-				"" = 64 // Reduce this number if you add things above. Make sure all the numbers in the list add to 100 EXACTLY 
+				"" = 64 // Reduce this number if you add things above. Make sure all the numbers in the list add to 100 EXACTLY
```

Comment:
VSC did this, go figure.

Reply:
Might be a good idea to change this empty string to `null`

Reply:
tested with null, causes runtimes.